src/elements/content-sidebar/SidebarNav.js

```diff
@@ -101,6 +105,15 @@ const SidebarNav = ({
                             <IconMetadataThick />
                         </SidebarNavButton>
                     )}
+                    {isDocGenTemplate && (
```

Comment:
Did we need to add tests for this in SidebarNav.test.js

Reply:
done